Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1719,9 +1616,9 @@ UpdateFspConfig (
     }
   }
 
-  if (TCC_FEATURE_ENABLED ()) {
-    TccModePostMemConfig (FspsUpd);
-  }
+  //if (TCC_FEATURE_ENABLED ()) {

```

Comment:
Why removing the Tcc check here? If no Tcc, we donot need to run this function at all.